config/site/src/_config.yaml

```diff
@@ -8,8 +8,8 @@ discord_url: https://discord.gg/8m9FqJ7a7F
 x_url: https://x.com/elasticgraph
 x_username: "@ElasticGraph"
 support_email: elasticgraph@squareup.com
-tagline: Schema-driven, scalable, cloud-native, batteries-included GraphQL with superpowers.
-description: ElasticGraph is an open-source framework for indexing, searching, and aggregating data powered by GraphQL and Elasticsearch / OpenSearch.
+tagline: ElasticGraph is an open-source framework for indexing, searching, and aggregating data powered by GraphQL and Elasticsearch / OpenSearch
+description: Schema-driven, scalable, cloud-native, batteries-included GraphQL with superpowers, backed by Elasticsearch / OpenSearch
```

Comment:
/nit to me, "ElasticGraph is an open-source framework..." seems more like a description and "Schema-driven, scalable..." more like a tagline. Maybe swap the names for these? (And update the usages to match?)